it/v6/lossClaim/amendType/def1/Def1_AmendLossClaimTypeISpec.scala

```diff
@@ -144,19 +147,26 @@ class Def1_AmendLossClaimTypeISpec extends IntegrationBaseSpec {
                               requestClaimId: String,
                               requestBody: JsValue,
                               expectedStatus: Int,
-                              expectedBody: MtdError): Unit = {
+                              expectedBody: MtdError,
+                              requestTaxYearClaimedFor: String = "2020-21",
+                              requestDownstreamTaxYearClaimedFor: String = "20-21"): Unit = {
         s"validation fails with ${expectedBody.code} error" in new Test {
 
           override val nino: String    = requestNino
           override val claimId: String = requestClaimId
 
+          override def uri: String =
+            s"/$nino/loss-claims/$claimId/tax-year/$requestTaxYearClaimedFor/change-type-of-claim"
+          override def ifsUrl: String =
+            s"/income-tax/claims-for-relief/$nino/$requestDownstreamTaxYearClaimedFor/$claimId"
+
```

Comment:
Rather than overriding the urls override the input data declared in the trait
`nino`, `claimId`, `requestTaxYearClaimedFor`